### DISCUSSION TITLE
BIOS files help where to place them

### DISCUSSION THREAD

**Discussion by paladinperry** [Rparadise-Team/Koriki#30] [General]
I am using a RG35XX using it in dual card setup

SD1: Koriki
SD2: Roms

I have the BIOS files in a BIOS folder on SD2 and SD1 and no success in launching PCEngineCD roms and other systems too
just blank screen then kicked to the menu

I love Koriki and have removed Garlic OS as theres just not enough settings and adjustability for me and I love Batocera
please can anyone help me with BIOS placement ?

**Comment by Ninoh-FOX** [comment]
Bios files is inside of the folder BIOS in the root of SDCARD.

**Comment by paladinperry** [comment]
Thanks for the reply.

But as in opening post, this is already present.

I have found out how to get PCengineCD ROMs to work now.

You manually change the BIOS used to "SuperGFX" from within Koriki itself NOT PCengineCD.

If anyone else is having this issue. That is how to remedy the issue.

Cheers
